.github/workflows/bump-version.yml

```diff
@@ -0,0 +1,132 @@
+name: Bump version
```

Comment:
So far, all templates have used `bump` so it is consistent. I would recommend to see a less familiar word in favor of something a clear word such as:

 ```suggestion
name: Increment version
```

It is consistent with how the official docs describes it:

<img width="854" alt="image" src="https://github.com/nimblehq/infrastructure-templates/assets/696529/61e4ef99-69e4-48d3-acb2-3bd97fc9fdab">

Bump has some unwanted meaning: https://www.urbandictionary.com/define.php?term=bump 👃 

> ['To](https://www.urbandictionary.com/define.php?term=%27To) Bump' something is to snort a drug. Usually [Ketamine](https://www.urbandictionary.com/define.php?term=Ketamine) but can be speed or coke.

> In message board terms, to reply to a topic that has become [inactive](https://www.urbandictionary.com/define.php?term=inactive), thus "[bumping](https://www.urbandictionary.com/define.php?term=bumping)" [the thread](https://www.urbandictionary.com/define.php?term=the%20thread) to the top of the thread list.

Reply:
I also prefer this name. Changed in https://github.com/nimblehq/infrastructure-templates/pull/237/commits/790a7d8f4c39523afe36a311082e94200773b2f7